config/routes.rb

```diff
@@ -4,7 +4,13 @@
   resource :user, only: [] do
     resources :timelogs 
   end
+  namespace :admin do
+    resources :users, only: [] do
+      resources :timelogs
+    end
+  end
   get 'pages/index'
+  post "admin/timelogs/download_pdf" => "admin/timelogs#download_pdf"
```

Comment:
why didn't you go with `respond_to format` solution?
This is not REST approach.